Makefile

```diff
@@ -54,23 +54,26 @@ version: ## output to version
 # Docker local development tasks
 ## Dargo == Docker Cargo
 dargo-prep: ## This runs everything neccessary to start developing locally in a container
-	# Spin up a long-running rust container
+	@# Spin up a long-running rust container
 	make dargo-create-container
 	# Compile and cache all dependencies
-	make dargo COMMAND=build
-	# Perform a first-sweep check, to fill up the cache (it seems to be different than the build one)
-	make dargo COMMAND=check
-    # Compile and cache test dependencies, run tests
-	make dargo COMMAND=test
+	@make dargo COMMAND=build
+	@# Perform a first-sweep check, to fill up the cache (it seems to be different than the build one)
+	@make dargo COMMAND=check
+	@# Compile and cache test dependencies, run tests
+	@make dargo COMMAND=test
 	# Now all caches are filled, and subsequent operations will be faster
 	# Consider adding dargo(){ make dargo COMMAND=$1} to your ~/.zshrc or ~/.bashrc for ergonomics
 
 dargo: ## Run a cargo command inside the container
+	@# If the dargo container does not exist, create it
+	@if [ -z $(shell docker ps | grep "$(LOCAL_DEV_CONTAINER_NAME)" | cut -c1-12) ]; then make dargo-prep; fi
+	@# Run a command inside the container
 	docker exec -ti $(LOCAL_DEV_CONTAINER_NAME) cargo $(COMMAND)
 
 dargo-create-container: ## Create a Rust container with this folder bind-mounted to it
```

Comment:
```suggestion
dargo-run-container: ## Runs a Rust container with cwd folder bind-mounted to it
```

Reply:
cwd == pwd?

Reply:
No, I really meant cwd as in "current working directory"

Reply:
Well, pwd is present working directory, and the command is pwd...

Reply:
According to `man pwd` stands for print current working directory, and `cwd` as a name is a thing in linux:
https://linux.die.net/man/3/cwd
I would read it as "Runs a Rust container with print current working directory folder bind...."
Well, actually `current working directory folder` doesn't reads much right as well ahah :D

Reply:
Lol gonna make it cwd to make you happy and be done with it